From 2a4fa179ff102ddb8efb8b29205f56580b90e897 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Tue, 15 Oct 2019 09:44:02 -0500
Subject: [PATCH] trivial: fwupd-refresh: allow exit code 2 for success

This is no supported devices, and shouldn't cause systemd failures
for fwupd-refresh.
---
 data/motd/fwupd-refresh.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/motd/fwupd-refresh.service.in b/data/motd/fwupd-refresh.service.in
index aa5d77d..bf82ed6 100644
--- a/data/motd/fwupd-refresh.service.in
+++ b/data/motd/fwupd-refresh.service.in
@@ -19,3 +19,4 @@ SystemCallFilter=~@mount
 ProtectKernelModules=yes
 ProtectControlGroups=yes
 RestrictRealtime=yes
+SuccessExitStatus=2
-- 
2.7.4

